models/reviews.models.js

```diff
@@ -81,3 +81,37 @@ exports.insertComment = (review_id, username, body) => {
       return result.rows[0];
     });
 };
+
+exports.alterVotes = (review_id, inc_votes) => {
+  if (!inc_votes || !/^-?[0-9]*$/.test(inc_votes)) {
+    return Promise.reject({
+      status: 400,
+      msg: "Invalid patch object",
+    });
+  }
+
+  return checkExists("reviews", "review_id", review_id)
+    .then(() => {
+      return db
+        .query(
+          `
+        SELECT votes FROM reviews WHERE review_id = $1;
+        `,
+          [review_id]
+        )
+        .then((result) => {
+          const newVotes = result.rows[0].votes + inc_votes;
+          return db.query(
+            `
+            UPDATE reviews SET votes = $1
+            WHERE review_id = $2 
+            RETURNING *;
+        `,
+            [newVotes, review_id]
+          );
+        });
+    })
```

Comment:
I appreciate this gets the job done - can you think of a way to get this done in a single query?
- you are able to reference table values from with a query